README.md

```diff
@@ -346,6 +328,38 @@ config:
 }
 ```
 
+**map[string]string**
+
+For a given configuration
+```go
+type Config struct {
+    headers map[string]string
+}
+```
+
+The values in the following examples will all be parsed as a string map of string values where the key and
+values get included as the string map key to string values.
+
+*yaml*
+```yaml
+config:
+  headers:
+    x-header-1: "a"
```

Comment:
suggestion: I'm for the feature but not so much in favor of using this as the example. All headers are naturally `map[string][]string` since it is RFC compliant to send the same header multiple times with multiples values.

Reply:
Good point.  I'll change it.